### DISCUSSION TITLE
2024 Akash Events Proposal

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#380] [Governance Proposals]
**Summary**

Akash Network participation at large In-person events are an important marketing, engineering, business development, community, investor relations and relationship building strategy. The following is a community proposal for resourcing Akash activities and participation at in-person events in 2024.

**TLDR : Proposal Development**

- Formed [2024 events working group](https://discord.com/channels/747885925232672829/1163891954652823612) 
- [2024 events proposal & budget drafted](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit#gid=0)
- [Community 2024 events proposal meeting](https://drive.google.com/file/d/1kGcp9cHtpcpUtbRrKJm3QZkZ78rwfa5V/view?usp=sharing) - 10/25 @ 10am PST
- [Community 2024 events proposal meeting](https://drive.google.com/file/d/1M1g7QnFcuw1q_69BF81ZS95aHpk6cP_G/view?usp=sharing) - 11/3 @ 11am PST
- [Proposal](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit#gid=0) Finalized

**Deep Dive : Proposal Development**

Event Selection
A broad list of known upcoming events were formed. The list went through a filter of AI/ML, Crypto, Cloud before being added to the review list. 

All of the above event types meet the prerequisites of 

- Established (Not the first time hosting event)
- Recognizable (Sponsors, attendees, and guests must be aligned with the Akash goals)

Once categories and prerequisites were established, the community reviewed the list of all proposed events for 2024 through a series of Working group calls

After community review, this left us with 16 total events to attend in 2024 including 2 overage events, e.g. - events we do not know about yet that may be of value for Akash to attend throughout the year.

**Event Participation**

Another consideration is who from the Akash community will attend these events. This will be decided case by case for each approved event listed in the proposal sheet. These community member selection discussions will happen openly in the [wg-events-2024 channel](https://discord.com/channels/747885925232672829/1111762722288316436) in Discord and in WG-Events calls.

Full definitions of roles, requirements and considerations can be found in the [Akash events: community support document](https://docs.google.com/document/d/16sjm4WpXfp4zZTQMJBKHQu3ySMQvENLoL9GzbmnJlfc/edit)

**Budget**

We defined event participation and costs into two categories.

Tier One events - These events will include a sponsorship to the main event and/or hosting an Akash Network owned side event 
Estimated Cost: $ 67,620.67/event

Tier Two events - These events will not include a sponsorship or an Akash owned meetup.
Estimated Cost: $32,174/event

**How things will get paid**

The budget will be put into a community wallet quarterly that Overclock labs will steward. The wallet information will be public and resources used for events will be reported out quarterly.

Members of the Overclock labs team that attend events will expense the cost of event attendance directly to Overclock Labs and Overclock Labs will return resources from the events wallet back to the company.

Akash community members that attend events will be given an estimated amount of AKT equal to USD for each event they attend as laid out in the [full proposal document here](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit#gid=0)

Forecasted Quarterly Breakdown of funding (USD)
Q1 : $167,415.69
Q2 : $99,795.02
Q3 : $131,174.35
Q4 : $32,174.35

The total resources allocated from the community pool will be $762,913.84 for the year 2024 However, this proposal will not request the entire sum all at once. Instead, resources will be taken out quarterly to ensure maximum accountability and to take into account potential price changes in AKT.
 
Additionally, each event will be reviewed individually in lead up to the event and before confirming tiered level (either 1 or 2). The total budget is just a forecast for what level of involvement Akash will want to have as of November 2023. In short, the total cost for all Akash events in 2024 should NOT go above the estimate above and will most likely be much lower than the total above request.

**Operational notes:** 
Overclock Labs will custody the requested funds in a new, distinct wallet so that funds from any other source are not commingled.
Responsible management of AKT received: the AKT funds received may not be liquidated immediately or not at all if market conditions are weak. If AKT is not sold, funding for events that require fiat or stables can be initially covered by Overclock Labs. If and when market conditions are strong, the right to liquidate these AKT to fund the events is reserved.
Overclock Labs will be stewards of the funding and at times cover the upfront costs of event participation in USD 

**Final Thoughts**

As mentioned in previous Akash community event proposals, from my experience running events for over a decade, I project the total cost for Akash to attend events to be much less than the proposed amount. Events are not a perfect science and each event will require different needs. But having a budget to work from will allow us to actively participate in community activities we will otherwise miss out on.

**Next Steps:**
- I'll leave this open for public discourse until **11/27/2023** at which time community feedback will be used to make any updates needed to the signaling proposal.
- If passed, the Events Working Group will create a Q1 2024 events proposal to go on-chain for a community vote.
- This process will continue each quarter throughout 2024

**Comment by piber-dev** [comment]
i believe that these events proposals are important for the network. i have personally attended one and knows how it works besides what is being said in the proposal and while i think that they can be improved, the improvements are not related to the proposal but only to the teams attending and the execution when being on spot. my only suggestion is that i would like to see some more interesting takes on the marketing collateral for the events where akash hosts workshops and booths. for example: helping people set up a wallet, create a deployment, and keep some ~10 AKT (maybe with authorized spend) to use for personal use. this could create users of akash easier than just mentioning what the project does.

**Reply by CaptainSplince** [reply]
Thanks @piber-dev - agreed on event activations. I think each event will have a different activations, some will be expo booth related others will be workshops and so on. Activations will be decided on before each event and will be case by case depending on the event.

**Comment by rodri-r** [comment]
As always, excellent description about your plans @CaptainSplince 

I agree with @piber-dev on these events bringing value and exposure to Akash.  I believe that this is necessary to grow the network.

**Reply by CaptainSplince** [reply]
Thanks fam!

**Comment by ronykris** [comment]
Great proposal @CaptainSplince !

I think these events make a lot of sense for the community and the ecosystem. The exchange of knowledge and vibes go a long way.

**Reply by CaptainSplince** [reply]
Thanks @ronykris. I agree!

**Comment by 31trainman** [comment]
Perhaps the akash team could look into setting up at big events that focus on cloud computing and such?

**Reply by CaptainSplince** [reply]
Thanks for the feedback. The events proposed have gone through the filter of cloud, crypto and AI, so I think they all touch cloud in one way or the other!

**Comment by miele101** [comment]
Hi I have a few questions:
Why is the TBA events have an estimated allocation amt that is more than the average price of the other 16 events? could this portion be approved separately from the confirmed event allocation. 
What is the spread of akt compensation for usd ? Would it be at a limit order after the event, when the proof of purchases for attending the event is made or would the participant not have any autonomy over at which price they are compensated?  
would you be able to host one of the TBA events in the east coast of the USA? Like New York or Miami for example which are two major crypto networking hubs. 
also I think we would all like to see the developments made from these investments. It would allow voters to get a better sense of what they are allowing the resources to be allocated towards. Thank you so much :)

**Reply by CaptainSplince** [reply]
Hey @miele101 - I'll try answering 1 by 1.

**Why is the TBA events have an estimated allocation amt that is more than the average price of the other 16 events? could this portion be approved separately from the confirmed event allocation.**

_I'm not sure your questions here.
There are 2 overage events (events we don't know exisit yet that we want to keep in a funding bucket just in case) 
There are 4 events we want to attend, but don't know the dates.
There are 10 events we know the dates of.
This will be a signalling proposal. Each quarter we will request that quarters budget for the events listed in that quarter and put unused funds back in the community pool_

**What is the spread of akt compensation for usd ? Would it be at a limit order after the event, when the proof of purchases for attending the event is made or would the participant not have any autonomy over at which price they are compensated?**

_I'll let my college who manges the AKT spend and payments answer this. @heyhewi_ 

**Would you be able to host one of the TBA events in the east coast of the USA? Like New York or Miami for example which are two major crypto networking hubs.**

_Potentially. It depends on what events we hear of and when._

**Also I think we would all like to see the developments made from these investments. It would allow voters to get a better sense of what they are allowing the resources to be allocated towards. Thank you so much :)**

_Sounds like you should join the WG-Events in our Discord channel. This is where we discuss each event and host retro calls. We also share the cost breakdown for each event. Beyond that, we host retro calls with all attendees where we share feedback and key learnings from each event._

Hope this helps.

**Reply by hewi333** [reply]
Hi there @miele101. Those from the community that were reimbursed for travel in 2023 were done so at the spot rate of AKT at the time they received reimbursement. You can also view the [2023 Dashboard](https://lookerstudio.google.com/u/0/reporting/c3273ae3-3749-46fe-8aae-4db6169cedf6/page/BIUND) for more information.

**Comment by shimpa1** [comment]
A great way to promote Akash! It has proven time and time again how important it is to bring the community together.
Europlots will vote yes on this prop.

**Reply by CaptainSplince** [reply]
Thank @shimpa1 !

**Comment by Aihonaken** [comment]
Love the events and i think the more we are involved in offline communication, the better. Would love to attend some of the events, but all in all, this is a great proposal, and happy that Akash community can be involved aswell.

**Reply by CaptainSplince** [reply]
Thanks @Aihonaken!

**Comment by avolon42x** [comment]
Great initiative - *invaluable* for the growth of Akash in 2024! Selection is great - hope we can make CFC happening as well!

Be aware: AirB&B is not always the cheapest accommodation - especially in Europe. Better to source through travel portals.

**Comment by hewi333** [comment]
Q2 Proposal is [live](https://www.mintscan.io/akash/proposals/254)!

# 2024 Events Q1 Budget Update

Actual cost: $161,797 (numbers are not final but should be close)
Amount off-ramped & awaiting off-ramp: ~$167,000
Amount of AKT leftover on wallet: ~$52,000 (at current price of $5.15 / AKT)

Q2 events requirement: $134,150.69
Q2 request: $82,150.69 (Q2 requirement - leftover AKT from Q1)
Q2 request in AKT (at spot rate of $5.15 / AKT): 15,951.59 AKT

It is important to note that with the volatility of AKT, any overages or underages will be taken into account in the subsequent quarter’s proposal.

2024 Events [Dashboard](https://lookerstudio.google.com/u/1/reporting/96c38594-7ec4-4a4f-9128-3feaf3b82ac8/page/BIUND)

**Reply by rodri-r** [reply]
Looks great and I believe events like Kubecon in Paris and others which were part of Q1 2024 events proposals were a great success. Drawing attention to the network, marketing and getting  more leads than initial goal.

**Comment by hewi333** [comment]
# 2024 Events end of Q2 Budget Update

Actual costs to date: $279,498.12 (numbers are not final but should be close)
Amount off-ramped to date: ~$218,985
Amount of AKT leftover on wallet to be liquidated: ~$47,664 (at current price of $3.23 / AKT)
Current Shortfall: ~$12,848 [Actual costs - (off-ramped + leftover on wallet)]

Q3 events requirements: $467,618.77
Q3 request: $480,467.43 (Q3 requirement + Current Shortfall)
Q3 request in AKT (at spot rate of $3.23 / AKT): 148,751.83 AKT. Rate will be adjusted at the time of submitting proposal.

It is important to note that with the volatility of AKT, any overages or underages will be taken into account in the subsequent quarter’s proposal. This is the first time there has been a shortfall since the community pool has been used for events dating back to Q2 2023.

The only major change for Q3 is that sponsorship amounts are going to $50k for tier 1 events from $30k. This is due to event sponsorship costs increasing throughout the industry. Also for operational reasons, the wallet recipient will change but will be accounted for in all calculations and will be captured on the dashboard.

2024 Events [Proposal](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit?gid=268068160#gid=268068160) with updated events.
2024 Events [Dashboard](https://lookerstudio.google.com/u/1/reporting/96c38594-7ec4-4a4f-9128-3feaf3b82ac8/page/BIUND)

**Reply by ghost** [reply]
Great, thanks for the update Hewi! 
If someone from the community would like to go and represent the Akash community in any of these events, feel free to fill out this form 

https://docs.google.com/forms/d/e/1FAIpQLSca0NL_KdebSUOgoS5MYSWofp1R2B_2pXKOKILUKHcBeig3pw/viewform

**Comment by hewi333** [comment]
# 2024 Events end of Q3 Budget Update

Actual costs to date: ~$797,499
Amount off-ramped to date: ~$687,980
Amount of AKT leftover on wallet to be liquidated: 0
Current Shortfall: ~$109,517 [Actual costs - (off-ramped + leftover on wallet)]

Q4 events requirements: $580,774.33
Q4 request: $690,291.33 (Q4 requirement + Current Shortfall)
Q4 request in AKT (at spot rate of $2.23 / AKT): 309,547.68 AKT.

It is important to note that with the volatility of AKT, any overages or underages will be taken into account in the subsequent quarter’s proposal. 

2024 Events [Proposal](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit?gid=268068160#gid=268068160) with updated events.
2024 Events [Dashboard.](https://lookerstudio.google.com/u/1/reporting/96c38594-7ec4-4a4f-9128-3feaf3b82ac8/page/BIUND)

[Q4 Proposal](https://www.mintscan.io/akash/proposals/272)

**Comment by brewsterdrinkwater** [comment]
The proposal for q4 events passed on chain: https://www.mintscan.io/akash/proposals/272

**Comment by SophoclesSummer** [comment]
This proposal was approved on 12/5/2023 with a 93% majority. 
https://www.mintscan.io/akash/proposals/239
